src/__tests__/to-be-visible.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Modal, View } from 'react-native';
+import { render } from '@testing-library/react-native';
+
+describe('.toBeVisible', () => {
+  test.each([
+    ['Empty view', <View testID="test" />],
+    ['View with opacity', <View testID="test" style={{ opacity: 0.2 }} />],
+    ['Modal', <Modal testID="test" />],
+    [
+      'View within modal',
+      <Modal>
+        <View testID="test" />
+      </Modal>,
+    ],
+  ])('handles positive test case: %s', (name, input) => {
+    const { getByTestId } = render(input);
+    expect(getByTestId('test')).toBeVisible();
+  });
+
+  test.each([
+    ['View with 0 opacity', <View testID="test" style={{ opacity: 0 }} />],
```

Comment:
```js
test.each([
    // ...
    ['View with display none', <View testID="test"  style={{ display: 'none' }} />],
    // ...
]);
```

To better mirror [`jest-dom`'s `toBeVisible`](https://github.com/testing-library/jest-dom#tobevisible) and cover additional contexts where an element would not be visible, WDYT about checking for `display: 'none'` as well?

Reply:
Good point, I overlooked the `display` prop for this case so I'll include it into the condition for checking the visibility.

Reply:
I added the suggestion in https://github.com/testing-library/jest-native/commit/e7afe068719863c95873c911a63e3f52c9004b46.